test/src/main/configurations/MainConfig/ConfigurationJavascript.xml

```diff
@@ -15,7 +15,7 @@
 					<sender className="nl.nn.adapterframework.senders.JavascriptSender"
 						jsFileName="JavaScript/JavascriptTest.js"
 						jsFunctionName="f2"
-						engineName="Rhino">
+						engineName="J2V8">
```

Comment:
I think this change should be in 7.9 proper already, and any other changes to the test-JS file needed to make it work (along with change to deprecate this configuration property).